spec/bolt_server/app_integration_spec.rb

```diff
@@ -60,7 +60,12 @@ def app
       expect(last_response.status).to eq(200)
       result = JSON.parse(last_response.body)
       expect(result).to include('status' => 'success')
-      expect(result['result']['_output'].chomp).to match(/\w+ got passed the message: Hello!/)
+      files = result['result']['_output'].split("\n").map(&:strip).sort
```

Comment:
This appears to have been a copy paste error, the expectation defined in the test would match the `echo` task, not the `shareable` task. We were not seeing this fail in CI because rspec filtered it out.